extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -443,4 +451,24 @@ public boolean isAnyPropertySet() {
         }
 
     }
+
+    /**
+     * Discriminator configuration.
+     *
+     * Separated in a group configuration, in case it is necessary to add the another existing hibernate discriminator property.
+     */
+    @ConfigGroup
+    public static class HibernateOrmConfigPersistenceUnitDiscriminator {
+        /**
+         * Existing applications rely (implicitly or explicitly) on Hibernate ignoring any DiscriminatorColumn declarations on
+         * joined inheritance hierarchies. This setting allows these applications to maintain the legacy behavior of
+         * DiscriminatorColumn annotations being ignored when paired with joined inheritance.
+         */
+        @ConfigItem(defaultValue = "false")
+        public Optional<Boolean> ignoreExplicitForJoined;
```

Comment:
You don't need to use `Optional` when you have a default value

Reply:
Done!